src/components/navBar/navBar.js

```diff
@@ -34,6 +34,9 @@ angular.module('material.components.navBar', ['material.core'])
  * listbox, according to
  * https://www.w3.org/TR/wai-aria-practices/#Site_Navigator_Tabbed_Style
  *
+ * In small screens that not support the entire width of this element, this directive works like md-tabs,
```

Comment:
nit: reword and remove reference to me-tabs. It doesn't add anything to the behavior for small screens. Something like:

If the width of the mdNavBar is greater than the available screen width, pagination will be used. Pagination supports left and right arrows keys as well as swipe gestures.